src_c/mixer.c

```diff
@@ -34,6 +34,13 @@
 
 #define PyBUF_HAS_FLAG(f, F) (((f) & (F)) == (F))
 
+#define CHECK_CHUNK_VALID(CHUNK, RET)                              \
+    if (CHUNK == NULL) {                                           \
+        PyErr_SetString(PyExc_RuntimeError,                        \
+                        "Sound object not initalised correctly."); \
+        return RET;                                                \
```

Comment:
I'd recommend wrapping `CHUNK` and `RET` in parenthesis. This is done because macro callers often expect the stuff they pass to be handled as normal expressions, and macro variable substitution can sometimes mess logic.

Not something that is a problem right now with the current uses of this macro, but could be in the future uses of this macro. 

Also while you are at it, it would also be cool if you could elaborate a bit more on that error message. Something mentioning that `__init__` was not called, so that people dealing with this error know exactly what to fix

Reply:
Those changers have been made. Hope the error message is better now.